server/src/test/java/org/opensearch/search/aggregations/bucket/terms/TermsAggregatorTests.java

```diff
@@ -353,33 +353,40 @@ private void testSimple(
                     newIndexWriterConfig().setMergePolicy(NoMergePolicy.INSTANCE)
                 )
             ) {
+                List<Document> documents = new ArrayList<>();
```

Comment:
nit: final?

Reply:
The variable has a very limited scope during index writing, so left it as non-final.